cargo-typify/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "cargo-typify"
+version = "0.0.11-dev"
+edition = "2021"
+
```

Comment:
what happens if you add default-run = cargo-typify here?

Reply:
That does exactly what we want I think!

Was easier than I was making it lol